state:
  teleoperation: 
    p: [0.0,0.0,0.0,0.0,0.0]
    i: [0.0,0.0,0.0,0.0,0.0]
    d: [10.0,10.0,100.0,100.0,100.0]
  robot_control:
    p: [2500.0,2500.0,5000.0,2500.0,2500.0]
    i: [2500.0,2500.0,5000.0,2500.0,2500.0] 
    d: [10.0,10.0,30.0,35.0,30.0]